wordpress-mobile/WordPress-iOS
# Posts & Pages: Pages context menu part 2

Closes #21873 

## Description
- Adds "set parent", "set as homepage", "set as posts page" actions to the page context menu
- Removes PostCardCell

## How to test
- Go to the posts list screen
- Look at PageMenuViewModel for test scenarios, and verify the expected menu items are displayed
- Test the "set parent", "set as homepage", "set as posts page" actions and verify they work as expected


https://github.com/wordpress-mobile/WordPress-iOS/assets/6711616/5e938e11-fc4c-4640-bc38-8c6056e13d2e



## Regression Notes
1. Potential unintended areas of impact
pages context menu

2. What I did to test those areas of impact (or what existing automated tests I relied on)
added more tests to PageMenuViewModel

3. What automated tests I added (or what prevented me from doing so)
see above

PR submission checklist:


UI Changes testing checklist:
